### DISCUSSION TITLE
Possible to download from multiple links?

### DISCUSSION THREAD

**Discussion by coolioniki** [nianeyna/ao3downloader#41] [Ideas]
Hello! This is an amazing piece of work.

I wanted to ask if there's a way to input multiple links (directly of the fic) into the program and download it that way? Like FanFicFare (which is amazing and what I have been using so far, but have also been having formatting issues with).

**Comment by nianeyna** [comment]
That's not currently possible no, but it's something I can certainly add! I've created an issue based on this discussion and placed it in the backlog with the "user requested" tag - I can't guarantee a timeline, but if you want to know when the feature is live you can subscribe to issue #42 in the issues tab. Thanks so much for using the discussion board!

**Reply by coolioniki** [reply]
Sounds good! Thank you!!
